.github/workflows/safe-transaction.yml

```
@@ -0,0 +1,87 @@
+name: 'Safe Transaction Proposer'
+on:
+  workflow_call:
+    inputs:
+      rpc-url:
+        description: 'RPC URL for the blockchain network'
+        required: true
+        type: string
+      chain-id:
+        description: 'Chain ID of the blockchain network (default: 42161 for Arbitrum)'
+        required: false
+        default: '42161'
+        type: string
+      safe-address:
+        description: 'Address of the Safe contract'
+        required: true
+        type: string
+      transaction-to:
+        description: 'Target address for the transaction'
+        required: true
+        type: string
+      transaction-value:
+        description: 'Value to send in the transaction (in wei, default: 0)'
+        required: false
+        default: '0'
+        type: string
+      transaction-data:
+        description: 'Transaction data/calldata (default: 0x)'
+        required: false
+        default: '0x'
```

There is no need to use this tool if there is no calldata. Transactions can be created directly from the Multisig UI.
```suggestion
        description: 'Transaction data/calldata'
        required: true
```